index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <img
+        src="https://svn.apache.org/repos/asf/openoffice/ooo-site/trunk/content/images/aoo-logo-100x100.png"
+        alt="logo"
+        height="50"
+        id="header-img"
+        width="50"
+      />
+      <nav id="nav-bar">
+        <a href="#form" class="nav-link">Jump to form</a>
+        <a href="#description" class="nav-link">Jump to description</a>
+        <a href="#footer" class="nav-link">Jump to end</a>
+      </nav>
+    </header>
+    <video
+      id="video"
+      src="https://www.w3schools.com/html/movie.mp4"
+      height="300"
+      width="500"
+      controls
+      autoplay
+    ></video>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <label for="email">Enter your email</label>
+      <input type="email" name="email" id="email" placeholder="Your email" />
+      <input type="submit" id="submit" value="Submit" />
+    </form>
+    <p id="description">
```

Comment:
- [ ] It would be great if you explain why you are requesting this change. 📜 Remember that a good review includes an explanation, a "why", for each requested change, this helps the code author understand how the change can improve the quality of the code. 🤓 

For example: 
> * This is a well-explained request: *Why don't you use an article tag for the `.first-article` element instead of a div, that would improve the HTML structure and make the page more accessible*
>* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

Please consider editing your comments to include an explanation! 🤓 🛠️